protected/humhub/modules/user/views/mails/plaintext/UserAlreadyRegistered.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/* @var string $passwordRecoveryUrl */
+?>
+<?= mb_strtoupper(Yii::t('UserModule.base', 'Welcome to %appName%', ['%appName%' => Yii::$app->name])) ?>
+
+
+<?= Yii::t('UserModule.base',
+    'You just tried registering at %appName% with this email address, but you already have an account connected to this email address.',
+    ['%appName%' => Yii::$app->name]) ?>
+
+<?= Yii::t('UserModule.base', 'Did you forget the password?') ?>
+
+<?= Yii::t('UserModule.base', 'If it wasn\'t you, just discard this message.') ?>

```

Comment:
`If this wasn\'t you, you can disregard this message.`

Reply:
Commit https://github.com/humhub/humhub/pull/6381/commits/055e40fdd7aabc6cde39fd046c0b182e3e906e3e